kouncil-frontend/libs/common-auth/src/lib/auth/kouncil-role.ts

```diff
@@ -1,6 +1,23 @@
 export enum KouncilRole {
```

Comment:
Not sure if its roles anymore. KouncilFeature?

Reply:
To keep everything similar between FE and BE called this enum `SystemFunctionName`